tf2_py/src/tf2_py.cpp

```diff
@@ -515,10 +532,11 @@ static PyObject *_allFramesAsDot(PyObject *self, PyObject *args, PyObject *kw)
 {
   tf2::BufferCore *bc = ((buffer_core_t*)self)->bc;
   static const char *keywords[] = { "time", NULL };
-  ros::Time time;
+
+  tf2::TimePoint time;
   if (!PyArg_ParseTupleAndKeywords(args, kw, "|O&", (char**)keywords, rostime_converter, &time))
     return NULL;
-  return PyString_FromString(bc->_allFramesAsDot(time.toSec()).c_str());
+  return stringToPython(bc->_allFramesAsDot(time).c_str());
```

Comment:
That method `PyString_FromString` didn't seem to exist for me. I don't know where it was coming from before.